packages/node_modules/@node-red/editor-client/src/js/ui/common/popover.js

```diff
@@ -84,6 +84,7 @@ RED.popover = (function() {
                 var targetHeight = target.outerHeight();
                 var divHeight = div.height();
                 var divWidth = div.width();
+                var paddingRight = 10;
```

Comment:
This does fix the one scenario you've highlighted, but I suspect similar issues will exist in all of the different orientations the popover could be used.

An alternative approach would be to use `div.outerWidth()` and `div.outerHeight()` in the lines above, so that the padding is accounted for there. But that would then require the `deltaSizes` magic numbers to be changed as they exist to counter-act padding.

To do this properly will require creating some tests (manual, not automated) that properly verify all of the different pieces of logic. I did have something for that when I wrote this code, but I can't find it now.

So on balance, I'm happy to merge this fix as-is for the given scenario as its the only *known* instance where the tooltip breaks.

Just noting for the future - some reworking will be need to get rid of the hardcoded numbers.